Cargo.toml

```diff
@@ -55,28 +55,26 @@ rmpv = "1.0.0"
 serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
 shlex = "1.1.0"
-simple_moving_average = "0.1.2"
 skia-safe = { version = "0.68.0", features = ["gl", "textlayout"] }
 spin_sleep = "1.1.1"
-strum = { version = "0.25.0", features = ["derive"] }
+strum = { version = "0.26.2", features = ["derive"] }
 swash = "0.1.8"
-time = "0.3.9"
+time = { version = "0.3.34", features = ["macros", "formatting"] }
 tokio = { version = "1.25.0", features = ["full"] }
 tokio-util = { version = "0.7.4", features = ["compat"] }
-toml = "0.7.3"
+toml = "0.8.12"
 tracy-client-sys = { version = "0.22.0", optional = true, default-features = false, features = ["broadcast", "delayed-init", "enable", "manual-lifetime", "fibers"] }
 unicode-segmentation = "1.9.0"
-which = "4.2.5"
-winit = { version = "=0.29.10", features = ["serde"] }
+which = "6.0.1"
+winit = { version = "=0.29.15", features = ["serde"] }
 xdg = "2.4.1"
 notify = "6.1.1"
 regex = "1.10.3"
 
 [dev-dependencies]
 approx = "0.5.1"
-mockall = "0.11.0"
```

Comment:
Good catch, it looks like this was not even used when it was introduced here:
* https://github.com/neovide/neovide/pull/261

Reply:
You could also consider running cargo-udeps in your CI, it caught some of the unused ones for me.